File: Signal.Delphi/Basic2.pas
unit Basic2;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  ExtCtrls, StdCtrls, ComCtrls;

type
  TBasicForm2 = class(TForm)
    Panel1: TPanel;
    Panel2: TPanel;
    Panel3: TPanel;
    RichEdit1: TRichEdit;
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    TimeCheck : DWord;
    TimeElapsed : DWord;
    { Public declarations }
  end;

var
  BasicForm2: TBasicForm2;

implementation

{$IFDEF CLR}
{$R *.nfm}
{$ELSE}
{$R *.dfm}
{$ENDIF}


procedure TBasicForm2.FormCreate(Sender: TObject);
begin
     RichEdit1.Lines.Clear;

end;

end.

File: Signal.CPP/Basic2.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Basic2.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TBasicForm2 *BasicForm2;
//---------------------------------------------------------------------------
__fastcall TBasicForm2::TBasicForm2(TComponent* Owner)
  : TForm(Owner)
{
  RichEdit1->Lines->Clear();
}
//---------------------------------------------------------------------------

